awscrt/mqtt.py

```diff
@@ -207,14 +205,26 @@ def suback(packet_id, topic, qos, error_code):
                     ))
 
         try:
-            assert callable(callback)
+            assert callable(callback) or callback is None
             assert isinstance(qos, QoS)
             packet_id = _awscrt.mqtt_client_connection_subscribe(self._binding, topic, qos.value, callback, suback)
         except Exception as e:
             future.set_exception(e)
 
         return future, packet_id
 
+    def subscribe_to_any(self, callback):
```

Comment:
I don't like this name because it implies that we're actually doing a SUBSCRIBE operation. How about on_any_publish or something? You could make it a set only property to make it clearer.